histomicstk/features/ComputeFSDs.py

```diff
@@ -0,0 +1,179 @@
+import collections
+import numpy as np
+from skimage.segmentation import find_boundaries
+
+
+def ComputeFSDs(Label, K=128, Fs=6):
+    """
+    Calculates `Fourier shape descriptors` from a label mask.
+
+    Parameters
+    ----------
+    Label : array_like
+        A M x N label mask.
```

Comment:
@slee172 Sanghoon - we can make each function accept a label image, and then perform the necessary looping over objects in the main function body. You can create a helper function to actually perform the calculation on each single object. This will make the code clean & manageable.